test/master_test.js

```diff
@@ -740,7 +740,7 @@ describe('Master tests', function() {
                         statistics: [{ 'Master': { 'errors': 0, 'invocations': 2 } }, { 'Collector': { 'errors': 1, 'invocations': 10 } }, { 'Updater': { 'errors': 0, 'invocations': 0 } }],
                         dl_stats: { dl_count: 6, max_dl_size: 4257 },
                         status: 'error',
-                        details: [errMessage],
+                        details: [errMessage.message],
```

Comment:
wouldn't this be `[JSON.stringify(errMessage)]`?

Reply:
no scott it would go into condition
else if(errStatus.message){
                        master._azureContext.log.warn('Health check failed with error message',  errStatus.message);
                        status = master.errorStatusFmt('ALAZU000005', errStatus.message);
                    }